docs/tools/wifi.sh

```diff
@@ -52,7 +52,8 @@ case "$os" in
 		if [[ ($input != y) && ($input != Y) ]]
 		then
 			echo "Removing the copy from the EFI partition"
-			sudo rm $mountpoint/wifi.tar.gz $mountpoint/asahi-installer-main.tar.gz $mountpoint/"$script_name"
+			sudo rm $mountpoint/wifi.tar.gz $mountpoint/asahi-installer-main.tar.gz
+			sudo rm $mountpoint/"$script_name" || (sudo rm $mountpoint/*.sh || (echo "Warning! Couldn't remove the copy of the script from the EFI partition. Please remove manually." && sleep 5))
```

Comment:
don't remove all .sh files, a user might have put other scripts there.

If the user is meant to remove the file manually, then pause the script until they press enter.

I'm not sure why we are adding this code to ensure the script is deleted. In what situation would it not be deleted by the current code? Do the other files need the same level of care to ensure they are deleted?

Reply:
For example :- I run wifi.sh on macOS. This will create a script named wifi.sh in EFI. Now, the user chooses to run the script itself again in Linux. Due to some reason, the script got renamed to, say wifi(1).sh. Now, `rm $mountpoint/"$script_name"` shall be `rm /tmp/apple-wifi-efi/wifi(1).sh`, causing error.

Reply:
411ea43

Reply:
Making it always save the copy of the script on the efi partition as "wifi.sh" and then always removing "wifi.sh" from the efi partition should prevent the possibility of the user having to manually remove it then

Reply:
and then we wouldn't have to worry about the commands to mount the efi partition and run the script from there changing